ansible/roles/baselayout/vars/main.yml

```diff
@@ -106,6 +106,6 @@ packages: {
   ],
 
   ubuntu1404: [
-    'ntp,g++-4.8,gcc-4.8,g++-4.9,gcc-4.9,binutils-2.26',
+    'ntp,g++-4.8,gcc-4.8,g++-4.9,gcc-4.9,binutils-2.26,gcc-6,g++-6',
```

Comment:
AFAICT GCC-8 is available, so that's preferable, as per https://github.com/nodejs/build/issues/1543, and latest discussion.

Reply:
Latest build WG discussion was 6.x is target, or so @mhdawson said. If there isn't consensus on that @refack, could you comment in #1543, to keep the discussion centralized?

Reply:
See my comment on #1543, Tegardless of whether GCC-8 is available we still need to test with GCC-6 to make sure we are supporting the minimum.  In the case of PPC we don't have enough existing machines to double up on the testing (which as discussed in the build wg meeting we already do for x86 so it is feasible there) to test GCC-6 and GCC-8 initially.  Therefore, for the 12.x release of PPC we will quite likely test and release on GCC-6.  There is some chance we can do better depending on how quickly we can get our centos machines up and running but I don't want to block or depend on that for 12.X.

Having said that, @sam-github if it's just a matter of adding gcc-8, and g++-8 to the list it might make sense to do that so that we have them installed for when we look to do more?

Reply:
The other thing I'll add is that we don't have the developer toolset for PPC for the existing machines and without that I'm not confident a binary built with 8 would run on a system that only has 6.x which is another thing we'd have to convince ourselves is safe.  With the developer tools (as we have on x86) I'm more convinced it will just work.  We may figure out the developer toolset on the centos machines for PPC  but again I'm not sure we will get that time for 12.x

Reply:
Ok. So lets land this as is, and I'll test compatibility of GCC-8 built binaries on a machine that does not have it.